index.html

```diff
@@ -5,14 +5,44 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" href="style.css">
     <title>API Practice</title>
 </head>
 
 <body>
-    <div class="container"></div>
-    <button id="random">Get Info</button>
-    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
-    <script src="main.js"></script>
+
+    <button id="random" onclick="getRickAndMorty()">Get Info</button>
+    <div class="container">
+
+    <ul>
+        <li class='item'>
```

Comment:
you can make it so that every item element inside a for loop, which creates a new element then adding the element into the html.

this would keep your HTML code cleaner